notebook/analysis/benchmark_results.ipynb

```diff
@@ -6340,6 +6340,2017 @@
h6fED31pGIMBZbxmzSyaZdfuED+9ZQSrEvDOg3hohkFLesVSqvoBml2BGm++9IK8W+TMhxpakFBoXigFHsCswlT2hcGupXE/yXe8vlchhjznsbJ8IUmmER9kpS8lMhad240wd2z/OW8GHd87rIvBPOWEdJKwKlaYhwMk5KMHJdOgq63SmlKG+DtpmRCKOxZd4lJTA7Qk05heVvOa0YCTQVl3St6tXr03lEqaQv+KWuOl0ul5mbmzvvbSIRKo2YfHvhM3FUY4tzjiGV1PIv97bZLTI9z+sOPqyvkUiy6E67DKZHK4aCdC0k5HWfRqsVaHtybFYp6iQlMTnftdhbIydJj/U558hpRVMcR5vCwQwUUxbY55zjWGSJW91fZrVmb6hS1fnJkEyCbwIFrVpfUfQaTTlc+bDcnpQbxzHOuc7HywX7rSjt8TxvY/iwvkZTseWuZkRdkj4LBuEqEfZk0jif3+sWptVdSERQrVaB7clnaTQRO2IrxM0mxoeHLdOQJOT2tFaVzCrFnHXMWpeqsG5FOBlZjErmE4gIs84xEcOu84TcbqOVYndoONq0zFiHkOyT4QtM+D3fpNyl2mF+NW8+2Hted0vPq1uXORJbZhyUTTITvmrh3qZlZxiksmWg1x3KWlFSmhknGOWIHQwEmkLK/qZEhO83Iu5pWpRS5JljxEZc01rsydtcskIP8rRFtFggEii3zi+UUmQU1NP2QICy1lyRSTqJKZIR9vVcxE1rjdb6nEm6y1ku2LdH79sj9+2PPc/bfD6sr9Gcg4xKyhQAlBZmnRC1VtD0vLUIlGJfxjBjHZEIWaPoMzp1qwGOWcsPmzF9RpE3AUFguKdWo1/DzlWMCnrrK6sURaWZc46CTkKvg9TVrBuVFIw0JWlHCe21CNL1/GjLa0W+C8rbLibY+9F5z9t8/qi5RkUF8y45UGig4pIX3nT3WvC6QaiS+Q9pVnegFGRbYTBrNCDU/Py4LWFaK/qeiqHmBKVgb2Aop6ybVaAUuwLNidhSt4JDUVCKwRStJpt27dVq29rzt+aAu6t1TBzTA6kbYPC8bubD+hrtCw3zTqjZJKwHCg6mcHl7r/tEImdH1lU6R9YDBU7AOoc2htg5HIowXQ9jkYWrf6ZRVisOZgJsq+wirY+lPzAIMOsEo2DY6NQvhpZmk9ZxIrbkjSESx6nIsVPDcMoHHDyvm/iwvkZDgeE6pRiLLY5kmfidga/H9S5NLMLxpmVOBKOE2EFFhD3r0Ed6M+0wmt2h4WTTEkhM2IjYFWh2pmwkF6DhhNPWUmu1PNwVakopKx9ZKO0dq2at43TsSKqnhcjBvky6usFsFyLChHXklCKvNQVjyCvFhLUMGe2Ph563TnxYXyOtFMOBYdBohPQfAL3uMOeEOXH0aoVSGqeSS8z9RlNK0d9YoDUPyYbsMIYGsKO3RE+tik5ZXWsswrHI0hBHViuiVsvDQxnVtat/bmdWhJOxJVBQ0goRmHWO8Rh2p6gbTNu8c1SsQ6Eod/GKsufjSK6ktWkFSDJ5OX2PxvO6U/pe3bpMmkY7ve4Xt0oU2iNSWikUgktXxgUg1JoDGY0xhv5CnqlGHWvtVm/WRamJUBVHX+uKQEYpZqyj4hx5nb7L/CJCTLIQTxoHGGJJ3pZ2g2mk8PkxbR3Ho/b1ARi3jgMZk6pWmkoperVizDoCLcQiVJyj36zPQlWe5yV8WPe8LpJr9Vafa422OQTN+rZ08y5NWvdEwyWj0tVWrfcOoxlIWV2xUclBqykQkuyLpgg9KRuRdiKcji0ZpTorE887x1jsKGbSE9YBdgQGB8yJEFjHoDEMp+sheF7X82Hd87pIQbV6lDdjrChCLTwgDDoHdG9z5VTSbWTOJit/RgIa1Vp1Mj3sgnKeglbEAidiS6AUPSmaRxAoxaBR3NmIqTmHKMVuoxlIWcC1gBXILtjsEEWUsjIxaHcaCnBa018sMBc1cM63ffK89ZSuV7gu40SoueTNpvBF1us+s06YcEJRK3pMEhZPW0clZeUj20WoFCOBIRbhdNNSiR27Qk0hRaUKAA0RauIoaYVRiqxWaJWM5qaJE6EGFHWyAujuQIOCWspefgMgrxTV1rEjdo66SOpOAhcKlSJsrZDred76uqxG1jOZDNlsdl3uKxbhWCNiOk5CVFEr9ucy5FZ5EA+CgHK5vC7bkhbtF/FisbhhC2g4ERzJwinddtBYzT4/Xa0Tx7ArE6CVwoowFsW4fJFy9sILlnSjzdjvG8WJMNWMyJuQskpGRBtak8tlCVcRrLrleW6sI19rkF/QBjSKLaXQUM5m1v3nbdQ+bzhHVG1wIH/2ccxbR2wU5dz6vLZfqtXu8ytiy9fnq5xqHUNGsgEHiwVyKbrS0Ra3Bq5mY0smX1g04dTzvEt3WYX1ZrNJs9lcl/s6FcWMWqE3Gdhh0gn1Wo2Dq1xOvVwuMzc3ty7bkhbGGDKZDJVKZUMmGk7FljPW4SRZoGokMF3Vf3k1+7zaiGg0YxouQrXq1xuxo6KFuWY6n64bvd83UsU5jjdjyjqZMBcAo1bINhsMrCJUdcvz3IkQxpbTVsgriBAQRWANc83Guv+8jdrnTRFqjRijz07urzmH1pq5aH1e2y/Vavf5sShGxY7dKplg2qjHnLRx6vqTN5xwLIppKE2pWCSuVdij9bqX7q3XQJvnpVH6TuG7RMUlBz2tkst+BZ1c0oy3esMuU3POcTy2aCCvoeocxyKbuvKkPqMpaMWcFeatMGuFXq1Ttyz8dtHuwrOws4VBiFPWnkcrxYjRFBTMWEcsMJLCcp4Q6DWKWZcE97oTYoHelD2OWIQ5K/SZpGVjj9EUtWbGpqssCeC0tUxZR8MlpVbTsetcLfA8b32k6xWuiwRKEXP2gB2TlF74X+jWmLcODeRaNbllo6mKo5HCsH6/Vvu2nE5WL70mE1BM4aXx7SCjFAZFoxXO41aZVRon/E47oSbS2faJWFI3oVEpxe7AMGiSSbIK2BuY1HWDUa
```

Comment:
My guess is that it's two opposing factors: accuracy gets worse as samples increase because the models are able to overfit the training data more, and the other side is probably related to the issue with the SKL LR implementation